src/pages/index.js

```diff
@@ -0,0 +1,261 @@
+//imports
+import "./index.css";
+import {
+  enableValidation,
+  config,
+  resetValidation,
+} from "../script/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+//api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3eb1ae88-bc1c-483c-97a8-5e99ad5e651d",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+    profileName.textContent = userInfo.name;
+    profileAvatar.src = userInfo.avatar;
+    profileDescription.textContent = userInfo.about;
+  })
+  .catch(console.error);
+
+//profile elements
+const profileAvatar = document.querySelector(".profile__avatar");
+const profileEditButton = document.querySelector(".profile__edit-btn");
+//form elements
+const editModal = document.querySelector("#edit-profile-modal");
+const profileName = document.querySelector(".profile__name");
+const editFormElement = document.querySelector(".modal__form");
+const editModalNameInput = document.querySelector("#profile-name-input");
+const profileDescription = document.querySelector(".profile__description");
+const editModalDescriptionInput = document.querySelector(
+  "#profile-description-input"
+);
+//deleteCard modal
+const deleteCardModal = document.querySelector("#deleteCard-modal");
+const cancelDeleteCardButton = deleteCardModal.querySelector(
+  ".modal__cancelDeleteCard-btn"
+);
+const deleteCardForm = deleteCardModal.querySelector(".modal__form");
+//avatarModal
+const avatarModal = document.querySelector("#edit-avatar-modal");
+const avatarModelButton = document.querySelector(".profile__avatar-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarForm = avatarModal.querySelector(".modal__form");
+//Card modal
+const cardModal = document.querySelector("#add-card-modal");
+const cardModelButton = document.querySelector(".profile__add-btn");
+const cardForm = cardModal.querySelector(".modal__form");
+
+//card modal elements
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+//card related elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+//selectmodal
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageElement = document.querySelector(".modal__image");
+const previewModalCaptionElement = document.querySelector(".modal__caption");
+// functions
+//add card modal
+function handleAddCardFormSubmit(evt) {
```

Comment:
* I can't add new cards or change the `avatar` because the page reloads after submission


Please, add `evt.preventDefault()` for any submit handler